web-client/src/pages/routes/DashboardRoute.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Location from 'react-app-location';
+import { Route, RouteProps } from 'react-router-dom';
+import { MenuItem } from 'src/types/menu-item';
+
+import DashboardLayout from '../../components/DashboardLayout/DashboardLayout';
+
+// TODO remove when using real profile state
+const mockProfileData = {
+  fullName: 'John Watkins',
+  photoUrl:
+    'https://lh3.googleusercontent.com/-6DkdLho7c5Y/AAAAAAAAAAI/AAAAAAAAAAA/AMZuuckYv30Z33APqrrucaWdEBbadI7WxQ/photo.jpg?sz=46',
+  followers: 15,
+  stars: 4.5,
+};
```

Comment:
I think here we'll fetch the profile state from redux rather than this mock. I've just checked the user model but left it to do later by a piece of advice.

Reply:
Sounds like a good idea.